README.md

```diff
@@ -42,6 +42,10 @@ python3 -m pip install pyracf
 
 * [Github Discussions](https://github.com/ambitus/pyracf/discussions)
 
+## Build Status
+
+![GitHub Actions Build Status](https://github.com/ambitus/pyracf/actions/workflows/.github-actions.yml/badge.svg?branch=dev)
```

Comment:
This build badge looks correct. My only suggestion is to move the build badge to the top of the README as follows instead of creating a build status section. Other open source projects follow a similar convention. I think the Tensorflow README is a good example to look at as an example: https://github.com/tensorflow/tensorflow/blob/master/README.md

```markdown
![pyRACF Logo](logo.png)

![GitHub Actions Build Status](https://github.com/ambitus/pyracf/actions/workflows/.github-actions.yml/badge.svg?branch=dev)

Python interface into the RACF management application programming interface.
```

Reply:
That makes sense, thank you for sharing the example of convention.